README.md

```diff
@@ -122,7 +122,7 @@ RUN set -x \
 	&& curl -fSL "https://github.com/krallin/tini/releases/download/$TINI_VERSION/tini.asc" -o /usr/local/bin/tini.asc \
 	&& export GNUPGHOME="$(mktemp -d)" \
```

Comment:
If this `GNUPGHOME` export is a best practice, it seems like it should be in the `python.tar.xz.asc` example below as well.